pyanaconda/ui/gui/spokes/advstorage/nvdimm.py

```diff
@@ -38,6 +38,9 @@
 PAGE_RESULT_ERROR = 2
 PAGE_RESULT_SUCCESS = 3
 
+NVDIMM_SUPPORTED_SECTOR_SIZES = ['512', '4096']
```

Comment:
Aren't we able to get these values from blivet?

Reply:
We are not.
@vojtechtrefny - would it make sense to provide such values ?

Reply:
After talking to Vojta I am just going to rename the constant.
There seems to be about 8 sizes supported for sector mode currently, but we don't want to offer all of them in UI. Vojta will probably propagate info about supported sizes by libblockdev api, and we may use it to check kickstart values, but it should be solved in a separate issue.

Reply:
Nice cooperation thanks!